scripts/ironic-common.sh

```diff
@@ -1,4 +1,22 @@
-export PROVISIONING_INTERFACE=${PROVISIONING_INTERFACE:-"provisioning"}
+function get_provisioning_interface() {
+  if [ -n "${PROVISIONING_INTERFACE}" ]; then
+    # don't override the PROVISIONING_INTERFACE if one is provided
+    echo ${PROVISIONING_INTERFACE}
+    return
+  fi
+
+  for mac in ${PROVISIONING_MACS//,/ } ; do
+    if ip -br link show up | grep -q "$mac"; then
+      export PROVISIONING_INTERFACE=$(ip -br link show up | grep "$mac" | cut -f 1 -d ' ')
```

Comment:
shouldn't this return after it finds the PROVISIONING_INTERFACE value ?

Reply:
it echo's the value at the end of the function